db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The 'bonus' field in the Skill model is defined as a CharField. If 'bonus' is meant to represent a numerical value, consider using an IntegerField or FloatField instead. This will ensure that the data type aligns with its intended use.